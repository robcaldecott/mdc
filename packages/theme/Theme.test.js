import React from "react";
import { Theme } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <Theme text="text-class" background="background-class">
        <p>Child</p>
      </Theme>
    )
  ).toMatchSnapshot();
});
