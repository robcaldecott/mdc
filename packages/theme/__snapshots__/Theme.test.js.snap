// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<p
  className="mdc-theme--text-class mdc-theme--background-class"
>
  Child
</p>
`;
